### DISCUSSION TITLE
Add a placeholder for loading images

### DISCUSSION THREAD

**Discussion by cdwmhcc** [directus/directus#12994] [Feature Requests]
## Loading...
![image](https://user-images.githubusercontent.com/8030134/165111313-42f8a1c9-c9f5-425c-9241-43b74d31feeb.png)

## Finish
![image](https://user-images.githubusercontent.com/8030134/165111349-2630ffba-9e37-4493-84da-d7c1e441441b.png)

Page flickers due to image loading.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
